index.html

```diff
@@ -1,15 +1,24 @@
 <!DOCTYPE html>
-<html>
+<html data-ng-app="WDIRadio">
   <head>
     <title>WDI Radio</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.2/angular.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.15/angular-ui-router.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.0-beta.2/angular-resource.min.js"></script>
     <script src="js/app.js"></script>
-
+    <script src="js/songs/songs.js"></script>
+    <script src="js/songs/index.controller.js"></script>
+    <!-- <script src="js/songs/new.controller.js"></script>
+    <script src="js/songs/edit.controller.js"></script>
+    <script src="js/songs/show.controller.js"></script>
+    <script src="js/songs/songs.factory.js"></script> -->
```

Comment:
You commented out your factory, so you can't make use of `.query` later on in your code.